pythonforandroid/recipes/mplfinance/__init__.py

```diff
@@ -0,0 +1,18 @@
+
+from pythonforandroid.recipe import PythonRecipe
+
+
+class MplfinanceRecipe(PythonRecipe):
+
+    # there is not an official release yet...
+    # so pinned to last commit at the time of writing
+    version = 'cde5008'
+    url = 'https://github.com/matplotlib/mplfinance/archive/{version}.zip'
```

Comment:
Jugging from PyPI there're some alpha releases we could use
```suggestion
    version = '0.12.3a3'
    url = 'https://pypi.python.org/packages/source/m/mplfinance/mplfinance-{version}.tar.gz'
```